pkgs/top-level/all-packages.nix

```diff
@@ -1530,15 +1530,15 @@ with pkgs;
 
   amule = callPackage ../tools/networking/p2p/amule { };
 
-  amuleDaemon = appendToName "daemon" (amule.override {
+  amuleDaemon = amule.override {
```

Comment:
lets convert these to `amule-daemon` and `amule-gui` and add a alias or a throw for them

Reply:
like this?

Reply:
yes